### PR TITLE
[WIP] Add rclone support

### PR DESCRIPTION
**Pull request by dene14**
Hi @rjkernick !

We've prepared alternative docker-nexus-backup image with RClone tool in order to support a variety of cloud object stores as a backend for backup.

We tested it with AWS, so would be great if someone can re-test with GCE's GS and provide some feedback.

In addition to that, since the change is not backward compatible in values.yaml we have to consider major version bump, need your opinion on that @rjkernick 

Please feel free to ping @MohamedZ3kri or @dene14, we really want to get that merged :)

### CODE REVIEW

**Comment by rjkernick** [comment]
Definitely a major version bump will be needed, which isn't really an issue since we have done it a few times now. Definitely push that change so the testing passes. I'll take a look through the rest this afternoon.

File: charts/sonatype-nexus/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: sonatype-nexus
-version: 3.6.0
+version: 4.0.0
 appVersion: 3.27.0
```
Comment (rjkernick) [inline charts/sonatype-nexus/Chart.yaml:4]: Please also update the sources with the new backup container project

**Comment by rjkernick** [comment]
Please update the maintainer email in the image as well: https://github.com/dbccompany/docker-nexus-backup/blob/master/Dockerfile

**Comment by rjkernick** [comment]
Failed linting on a couple things:

charts/sonatype-nexus/values.yaml
  158:18    error    trailing spaces  (trailing-spaces)
  159:18    error    trailing spaces  (trailing-spaces)
  187:7     error    wrong indentation: expected 4 but found 6  (indentation)

**Comment by dene14** [comment]
@rjkernick fixed

**Review by rjkernick — APPROVED** [review]

**Pull request merged**